src/model/update.reset.ts

```diff
@@ -0,0 +1,29 @@
+
+import { ApplicationData } from "./application.model";
```

Comment:
I'd move this file under /utils as it's more a utility function to reset the model rather than the model itself.

Reply:
done